composeApp/src/commonMain/kotlin/sp/bvantur/tasky/register/ui/RegisterScreen.kt

```diff
@@ -0,0 +1,138 @@
+package sp.bvantur.tasky.register.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import org.jetbrains.compose.resources.stringResource
+import sp.bvantur.tasky.core.ui.components.BackButton
+import sp.bvantur.tasky.core.ui.components.ConfirmationButton
+import sp.bvantur.tasky.core.ui.components.PasswordTextField
+import sp.bvantur.tasky.core.ui.components.TitleText
+import sp.bvantur.tasky.core.ui.components.UserDataTextField
+import sp.bvantur.tasky.core.ui.components.UserOnboardingSurface
+import tasky.composeapp.generated.resources.Res
+import tasky.composeapp.generated.resources.create_your_account
+import tasky.composeapp.generated.resources.email_address
+import tasky.composeapp.generated.resources.get_started
+import tasky.composeapp.generated.resources.name
+import tasky.composeapp.generated.resources.password
+
+@Composable
+fun RegisterRoute() {
+    RegisterScreen()
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun RegisterScreen() {
+    val (emailRequester, passwordRequester) = FocusRequester.createRefs()
```

Comment:
Same here, wrap the instance creation with `remember{}` to preserve the state across recompositions.

```
val (emailRequester, passwordRequester) = remember { FocusRequester.createRefs() }
```